Dockerfile

```diff
@@ -9,11 +9,12 @@ WORKDIR /app
 
 FROM base as pydeps
 
-RUN pip install "poetry==1.7.1" && \
-    python -m venv ${VIRTUAL_ENV}
-COPY pyproject.toml poetry.lock ./
-RUN poetry install --only=main --no-interaction
-
+RUN pip install uv
```

Comment:
Can you change this to the officially recommended installation method?
https://docs.astral.sh/uv/guides/integration/docker/#installing-uv